gpsoauth/__init__.py

```diff
@@ -50,18 +54,19 @@ def init_poolmanager(self, *args, **kwargs):
         context.options |= ssl.OP_NO_COMPRESSION
         context.options |= ssl.OP_NO_SSLv2
         context.options |= ssl.OP_NO_SSLv3
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
This is added in Python 3.8: https://docs.python.org/3/library/ssl.html#ssl.SSLContext.post_handshake_auth

Should we set minimum supported version to 3.8? We use it for Home Assistant component which has minimum supported version 3.8 and this will allow me to use some modern features here.

Reply:
good catch; I didn't realize that. I guess we should.